File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/clis/CLIS2015U04GetPatientStatusListItemInfo.java
/**
 * 
 */
package nta.med.data.model.ihis.clis;

import java.util.Date;

public class CLIS2015U04GetPatientStatusListItemInfo {
	private Integer patientStatusId;
	private String description;
	private Date updated;
	private Date updateDate;
	private String codeName;
	private Integer sortNo;
	private String sysId;
	private String code;
	private Integer protocolPatientId;
	
	public CLIS2015U04GetPatientStatusListItemInfo(Integer patientStatusId,
			String description, Date updated, Date updateDate, String codeName, Integer sortNo,
			String sysId, String code, Integer protocolPatientId) {
		super();
		this.patientStatusId = patientStatusId;
		this.description = description;
		this.updated = updated;
		this.updateDate = updateDate;
		this.codeName = codeName;
		this.sortNo = sortNo;
		this.sysId = sysId;
		this.code = code;
		this.protocolPatientId = protocolPatientId;
	}
	public Integer getPatientStatusId() {
		return patientStatusId;
	}
	public void setPatientStatusId(Integer patientStatusId) {
		this.patientStatusId = patientStatusId;
	}
	public String getDescription() {
		return description;
	}
	public void setDescription(String description) {
		this.description = description;
	}
	public Date getUpdated() {
		return updated;
	}
	public void setUpdated(Date updated) {
		this.updated = updated;
	}
	public String getCodeName() {
		return codeName;
	}
	public void setCodeName(String codeName) {
		this.codeName = codeName;
	}
	public Integer getSortNo() {
		return sortNo;
	}
	public void setSortNo(Integer sortNo) {
		this.sortNo = sortNo;
	}
	public String getSysId() {
		return sysId;
	}
	public void setSysId(String sysId) {
		this.sysId = sysId;
	}
	public Date getUpdateDate() {
		return updateDate;
	}
	public void setUpdateDate(Date updateDate) {
		this.updateDate = updateDate;
	}
	public String getCode() {
		return code;
	}
	public void setCode(String code) {
		this.code = code;
	}
	public Integer getProtocolPatientId() {
		return protocolPatientId;
	}
	public void setProtocolPatientId(Integer protocolPatientId) {
		this.protocolPatientId = protocolPatientId;
	}
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Clis/CLIS2015U04GetPatientStatusListItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Clis
{
    [Serializable]
    public class CLIS2015U04GetPatientStatusListItemInfo
    {
        private String _patientStatusId;
        private String _description;
        private String _updated;
        private String _codeName;
        private String _sortNo;
        private String _sysId;
        private String _updateDate;
        private String _code;
        private String _protocolPatientId;
        private String _rowState;

        public String PatientStatusId
        {
            get { return this._patientStatusId; }
            set { this._patientStatusId = value; }
        }

        public String Description
        {
            get { return this._description; }
            set { this._description = value; }
        }

        public String Updated
        {
            get { return this._updated; }
            set { this._updated = value; }
        }

        public String CodeName
        {
            get { return this._codeName; }
            set { this._codeName = value; }
        }

        public String SortNo
        {
            get { return this._sortNo; }
            set { this._sortNo = value; }
        }

        public String SysId
        {
            get { return this._sysId; }
            set { this._sysId = value; }
        }

        public String UpdateDate
        {
            get { return this._updateDate; }
            set { this._updateDate = value; }
        }

        public String Code
        {
            get { return this._code; }
            set { this._code = value; }
        }

        public String ProtocolPatientId
        {
            get { return this._protocolPatientId; }
            set { this._protocolPatientId = value; }
        }

        public String RowState
        {
            get { return this._rowState; }
            set { this._rowState = value; }
        }

        public CLIS2015U04GetPatientStatusListItemInfo() { }

        public CLIS2015U04GetPatientStatusListItemInfo(String patientStatusId, String description, String updated, String codeName, String sortNo, String sysId, String updateDate, String code, String protocolPatientId, String rowState)
        {
            this._patientStatusId = patientStatusId;
            this._description = description;
            this._updated = updated;
            this._codeName = codeName;
            this._sortNo = sortNo;
            this._sysId = sysId;
            this._updateDate = updateDate;
            this._code = code;
            this._protocolPatientId = protocolPatientId;
            this._rowState = rowState;
        }

    }
}